agent/xdsv2/rbac_resources.go

```diff
@@ -41,7 +41,7 @@ func MakeL4RBAC(defaultAllow bool, trafficPermissions *pbproxystate.TrafficPermi
 	}
 
 	// Only include the allow RBAC when Consul is in default deny.
-	if includeAllowFilter(defaultAllow, trafficPermissions) {
+	if !trafficPermissions.DefaultAllow {
```

Comment:
🤔 what if I want to do the following in default-allow mode:
1. create an allow-nothing traffic permission in a partition
2. create an allow permission to a specific destination

Reply:
I think I misunderstood that this means default allow only if there are no traffic perms thinking that this is a global default. Maybe a comment here would be helpful